src/battle_script_commands.c

```diff
@@ -1250,7 +1250,9 @@ static void Cmd_attackcanceler(void)
     u32 i;
     for (i = 0; i < gBattlersCount; i++)
```

Comment:
```suggestion
    for (i = 0; i < gCurrentTurnActionNumber; i++)
```
Small optimization (only check battlers that have already acted)